src/css/support.js

```diff
@@ -38,6 +44,8 @@ define( [
 
 		var divStyle = window.getComputedStyle( div );
 		pixelPositionVal = divStyle.top !== "1%";
+
+		// Support: Android 4.0 - 4.3 only, Firefox <=3 - 44+
```

Comment:
Ah, OK then. I didn't seem the whole context on mobile so I thought this
line is new. :)

-- 
Michał Gołębiowski